src/main/java/com/training/services/ga/validate/RequestBody.java

```diff
@@ -5,4 +5,21 @@
 @Setter
 public class RequestBody {
     String email;
+    String vdsId;
```

Comment:
Why is the loyalty pojos under - com.training.services.ga.validate?
This should be under - com.training.services.ga.loyalty